packages/grafana-data/src/types/explore.ts

```diff
@@ -0,0 +1,20 @@
+import { ExploreMode } from './datasource';
+import { RawTimeRange } from './time';
+import { LogsDedupStrategy } from './logs';
+
+export interface ExploreUrlState {
```

Comment:
Moved here from explore code to prevent import cycles

Reply:
That's because of `mapInternalLinkToExplore` becoming part of grafana-data as for now?